docs/README.md

```diff
@@ -8,6 +8,12 @@ The first blockchain in the Cosmos Network is the Cosmos Hub, whose native token
 
 Cosmos can interoperate with multiple other applications and cryptocurrencies. By creating a new zone, you can plug any blockchain system into the Cosmos hub and pass tokens back and forth between those zones, without the need for an intermediary.
 
+## Quick Start
+
+- [Getting started with the SDK](./sdk/core/intro.html)
+- [SDK Examples](../examples)
+- [Join the testnet](./getting-started/full-node.html#run-a-full-node)
```

Comment:
`.md` otherwise this won't work on GH

Reply:
Change the `.html` endings to `.md`, do you mean? Won't that break on the web view?

Reply:
no, unlike RTD, vuepress handles this

Reply:
as proof, there's at least two "internal" links in: https://github.com/tendermint/tendermint/blob/master/docs/app-dev/getting-started.md

one uses `.html` and it doesn't work on GH but works on Vue
another uses `.md` works on both GH and Vue